daru.gemspec

```diff
@@ -69,4 +70,6 @@ EOF
   spec.add_development_dependency 'activerecord', '~> 4.0'
   spec.add_development_dependency 'sqlite3'
   spec.add_development_dependency 'rubocop', '>= 0.40.0'
+  spec.add_development_dependency 'simplecov'
+  spec.add_development_dependency 'nokogiri'
```

Comment:
As it been a **development** dependency, and as it been a "first default" XML parser, I think we should just "use the simplest option". And replace it on the very first complain of other developers.

Of course, I'd never thoughtlessly add it as a runtime dep.